storage/s3_test.go

```diff
@@ -1295,3 +1295,58 @@ func (e tempError) Error() string { return e.err.Error() }
 func (e tempError) Temporary() bool { return e.temp }
 
 func (e *tempError) Unwrap() error { return e.err }
+
+func TestS3HeadObject(t *testing.T) {
```

Comment:
keep test functions above the helpers, please move it right after `TestAWSLogLevel`.